lambdas/data-migration2/tests/test-pdrs.js

```diff
@@ -25,6 +25,7 @@ const {
 } = require('@cumulus/aws-client/S3');
 const { dynamodbDocClient } = require('@cumulus/aws-client/services');
 const { RecordAlreadyMigrated, PostgresUpdateFailed } = require('@cumulus/errors');
+const { getExecutionUrlFromArn } = require('@cumulus/message/Executions');
```

Comment:
We have it imported here but I'm not sure if we need it here since the factory is handling this for us now? Regardless linting will fail if it's not needed here

Reply:
https://github.com/nasa/cumulus/pull/2390/commits/c1426a9975c50a4d15b75941a87e91e5cf4f9a7e